resources/locales.clj

```diff
@@ -1,6 +1,9 @@
 {:locales
  #{"nl"
    "en"
+   "sq"
```

Comment:
Let's not mess with the `locales.clj` :)
Please checkout and remove this file from the PR.